website/server/libs/taskManager.js

```diff
@@ -113,7 +113,7 @@ export async function createTasks (req, res, options = {}) {
 
     // Otherwise update the user/challenge/group
     if (!taskOrderToAdd[`${taskType}s`]) taskOrderToAdd[`${taskType}s`] = [];
-    taskOrderToAdd[`${taskType}s`].unshift(newTask._id);
+    if (!owner.tasksOrder[`${taskType}s`].includes(newTask._id) & !taskOrderToAdd[`${taskType}s`].includes(newTask._id)) taskOrderToAdd[`${taskType}s`].unshift(newTask._id);
```

Comment:
I think this is missing a second &, but I don't think the second check is necessary (the one for taskOrderToAdd[`${taskType}s`])

Reply:
Sure, Maybe I was fussing over it, I've deleted it.